R/get_metadata_gee.R

```diff
@@ -1,78 +1,72 @@
 #' Helper function: convert to list
 #'
-#' @param coords column with satellite metadata coordinates (system:footprint`$coordinates)
-#'
+#' @param coords column with satellite metadata coordinates
+#'   (system:footprint`$coordinates)
 #' @return list of coordinates
 #' @keywords internal
 #'
-convert_to_list <- function(coords) {
-
-  tmp_mat <- lapply(seq_along(coords),
-                    FUN = function(idx) {
-
-                      t(as.matrix(coords[[idx]]))
-                    })
-
-  list(do.call(rbind, tmp_mat))
+convert_to_list <- function(coords)
+{
+  lapply(coords, function(x) t(as.matrix(x))) %>%
+    do.call(what = rbind) %>%
+    list()
 }
 
-
 #' Google Earth Engine: get metadata for collection
 #'
 #' @param collection collection
-#'
 #' @return tibble with metadata. In addition also the complex column
-#' "geometry_meta_org" is simplified and stored in the new column "geometry_meta_cleaned"
+#'   "geometry_meta_org" is simplified and stored in the new column
+#'   "geometry_meta_cleaned"
 #' @export
 #' @importFrom tibble as_tibble tibble
 #' @importFrom dplyr bind_cols bind_rows
+#' @importFrom kwb.utils removeColumns
 #' @importFrom tidyr nest
-#' @importFrom tidyselect matches all_of
-gee_get_metadata <- function(collection) {
-
-  nImages <- collection$size()$getInfo()
+#' @importFrom tidyselect matches
+gee_get_metadata <- function(collection)
+{
+  metadata_list <- collection$size()$getInfo() %>%
+    collection$toList() %>%
+    collection$fromImages()
 
-  collectionList <- collection$toList(nImages)
-
-  metadata_list <- collection$fromImages(collectionList)
   metadata_list <- metadata_list$getInfo()
```

Comment:
`metadata_list$getInfo()$features` da `metadata_list <- metadata_list$getInfo()` gelöscht wurde

Reply:
Hm. Steht das nicht in Zeile 33?